### DISCUSSION TITLE
My Helm Chart using Nginx Ingress controller for DigitalOcean Kubernetes

### DISCUSSION THREAD

**Discussion by sonamsamdupkhangsar** [digitalocean/Kubernetes-Starter-Kit-Developers#132] [Show and tell]
I wanted to share my Helm chart that I built which uses Nginx Ingress controllers.  
My Helm chart is located here https://github.com/sonamsamdupkhangsar/sonam-helm-chart if you are interested.  I also wrote a medium sized article on[ medium](https://medium.com/@sonamhava/deploy-to-digitalocean-kubernetes-nginx-ingress-controller-with-a-helm-chart-6796bfd11c4d) itself.

**Comment by vladciobancai** [comment]
Hi @sonamsamdupkhangsar we will review it and we will let you know our feedback
